docs/hub/spaces-url-query-hash.md

```diff
@@ -0,0 +1,16 @@
+# Set URL query and hash
+
+On a Space page (`https://huggingface.co/spaces/<user>/<app>`), the actual application page (`https://*.hf.space/`) is embedded in an iframe. The query string and the hash attached to the parent page URL are propagated to the embedded app on initial load, so the embedded app can read these values without special consideration.
+
+In contrast, updating the query string and the hash of the parent page URL from the embedded app is not straightforward.
+If you want to do this in a Static or Docker Space, you need to write JS code like this that sends a message to the parent page that has a `queryString` and/or `hash` key.
+
```

Comment:
```suggestion
You only need to do this for Docker or static Spaces: for Gradio and Streamlit apps, this is already done for you inside the SDK.
```

Reply:
Precisely, this is only done for Streamlit but not for Gradio because it doesn't have query/hash-related features.
Will add this sentence with this fix.

Reply:
Would be awesome to add this to Gradio no? I think it's planned no? cc @abidlabs